salt/pillar/varstack_pillar.py

```diff
@@ -21,27 +21,21 @@
 
 from __future__ import absolute_import, print_function, unicode_literals
 
-# Import python libs
-import logging
 
-HAS_VARSTACK = False
 try:
     import varstack
-    HAS_VARSTACK = True
 except ImportError:
-    pass
+    varstack = None
 
-# Set up logging
-log = logging.getLogger(__name__)
 
 # Define the module's virtual name
 __virtualname__ = 'varstack'
 
 
 def __virtual__():
-    if not HAS_VARSTACK:
-        return False
-    return __virtualname__
+    if varstack:
+        return __virtualname__
+    return False, 'The varstack_pillar module could not be loaded: varstack module not found.'
```

Comment:
You do not need this. Please return just this:

```python
return bool(varstack), '......
```
Your module is not any virtual, just True is totally sufficient. In case `varstack` is `None`, the error message next to it will be reused.